src/test/emulators/extensionsEmulator.spec.ts

```diff
@@ -0,0 +1,9 @@
+import { expect } from "chai";
+
+import { ExtensionsEmulator } from "../../emulator/extensionsEmulator";
+
+describe("Extensions Emulator", () => {
+  describe("getEmulatableBackends", () => {
+    // TODO: Write some tests for this (or maybe just write functional tests?)
```

Comment:
Working on these now, just a ironing out a few final details. I want to include these in this PR